integrations/adobe-analytics/test/chromecast.index.test.js

```diff
@@ -0,0 +1,491 @@
+'use strict';
+
+var Analytics = require('@segment/analytics.js-core').constructor;
+var integration = require('@segment/analytics.js-integration');
+var sandbox = require('@segment/clear-env');
+var tester = require('@segment/analytics.js-integration-tester');
+var AdobeAnalytics = require('../lib');
+var iso = require('@segment/to-iso-string');
+
+describe('Adobe Analytics - Chromecast', function () {
+  var analytics;
+  var adobeAnalytics;
+  var user;
+  var options = {
+    version: 2,
+    chromecastToggle: true,
+    reportSuiteId: 'sgmtest',
+    trackingServerUrl: 'exchangepartnersegment.sc.omtrdc.net',
+    trackingServerSecureUrl: '',
+    heartbeatTrackingServerUrl: 'https://exchangepartnersegment.hb.omtrdc.net',
```

Comment:
update to `heartbeatTrackingServerUrl: 'https://exchangepartnersegment.hb-api.omtrdc.net',` for tests to work

Reply:
@Isuru-F `index.test.js` unit tests are failing, but I'm not sure those matter since they are for the JS (non-Chromecast) SDK?